### ISSUE TITLE
 Species list in a dropdown menu for unknown species

### ISSUE THREAD

**Issue by JotablancoA** [BiologicalRecordsCentre/MammalNet-Europe#46]
Although this is not a priority. I'm curious to know if for the species we don't have cards in the app and we have to categorize them as unknown species, Would it be possible to incorporate in the app a drop-down menu with the potential species that could be selected by group? 
Downloading a file of scientific names (and common???) does not have to be very heavy. 
For example if I select Mole in the app, that it  gives me the possibility to indicate it as unknown or select one of the scientific names available for mole in Europe (with a dropdown menu). 
It could be a cheap and simple way to cover more species. 
Best,

**Comment by kazlauskis** [comment]
Hi @JotablancoA, I can definitely add the scientific names for a start. Later after everyone translates all the common species names I could add them in as well.

If someone provided me the list of extra species in such format then I could start the work on it. Cheers!

english | taxon | group
-- | -- | --
Fluffy mole | Atellius labatinus | mole

**Comment by JotablancoA** [comment]
Hi,

If David (in CC) gives us the complete list of European species used by
Indicia in website we can perform the translation.
Perhaps Grahama (in CC) want to organice this task

**Comment by DavidRoy** [comment]
The Indicia list is given in another issue: #30 

If you can extend by adding in the common names from other languages, we can update this master-list

**Comment by kazlauskis** [comment]
To add the most basic functionality (scientific names only) to the app this would need a species list where latin names are mapped to the unknown species groups from the app. Until then I am blocked on this one. 

This is how the ideal look like:


id | english | taxon | group
-- | -- | -- | --
452353 | Fluffy mole | Atellius labatinus | mole

**Comment by JotablancoA** [comment]
Let me know, if i am wrong. This file in  issue: #30
<https://github.com/BiologicalRecordsCentre/MammalNet-Europe/issues/30> is
not complete, there are missing species, as mustelids, chiropters
...others.

I had understood that users could enter any european mammal species name
from the app, including chiroptera, cetacea,...am I right?

Although it was proposed to be able to locate with the drop-down menu those
species that did not have a card in the app. Perhaps it is more appropriate
to offer the user the possibility of entering in a searcher any species by
its scientific or common name. Ideally, as you introduce letters, you
reduce the possibilities. In a similar way to the web guide
https://mammalnet.com/filters/.

Best,

Jose A Blanco

**Comment by DavidRoy** [comment]
@GrahamCSmith can you add the extra taxa when adding the common names to the list
@JotablancoA we could add extra functionality for name searching but this was not part of the original specification.  Is there potential for additional funding to implement this?  I agree that now the list is longer, a search facility is needed

**Comment by JotablancoA** [comment]
Hi, 
"name searching but this was not part of the original specification "
I probably misunderstood what you meant, or  I made a poorly specified proposal. When i did it in January was just a question to asses. 
I don't know what @kazlauskis will think, but I think it's easier and less confusing for the user to implement a search engine by name for all species, than a drop-down menu for 10 different unknown species categories.  Mainly to upload DB with a complete list of names and link to search engine. 
The person who knows the names of the species doesn't need the images. The idea of this proposal is to attract more experienced users, who are usually the ones who participate the most. 
illustrations (unknown species) could be left for less experienced users and a single search option for more experienced users.  In this way, I think it is  it is possible to save time. 
Please, let me know if I'm still wrong about the efficiency of this proposal.

**Comment by DavidRoy** [comment]
Thanks. I understand you were just asking to assess possibilities.  I agree this enhancement would be beneficial.

Development wise, it is ~3 days effort to add a search/filter/dropdown functionality to the app.

**Comment by GrahamCSmith** [comment]
I'm not sure if we should proceed with this at this stage as it will need a full species list (eg. bats, rodents and shrews) for each country. We could add this functionality after teh EMMA2 atlas has been completed?
